pkg/controlplane/apiserver/options/options.go

```diff
@@ -262,8 +266,39 @@ func (o *Options) Complete(alternateDNS []string, alternateIPs []net.IP) (Comple
 			if err != nil {
 				return CompletedOptions{}, fmt.Errorf("failed to build token generator: %w", err)
 			}
+		case completed.ServiceAccountSigningEndpoint != "":
+			plugin, cache, err := plugin.New(ctx, completed.Authentication.ServiceAccounts.Issuers[0], completed.ServiceAccountSigningEndpoint)
+			if err != nil {
+				return CompletedOptions{}, fmt.Errorf("while setting up external-jwt-signer: %w", err)
+			}
+			metadata, err := plugin.GetServiceMetadata(ctx)
```

Comment:
Add an integration test that shows that we can tolerate the plugin starting significantly after the API server has started running, i.e. ~30 seconds.

Reply:
How is that? Api Server needs to pre-fill supported keys to start. Options will not be completed unless External Signer is ready.

Reply:
what is the behavior here if the external signer is initializing along with the kube-apiserver? does the plugin try to connect a single time on grpc client construction and fail instantly if the backend is not available? does it keep retrying internally? for how long?

Reply:
a single getmetadata call is fine... I thought the gprc dial / conn construction would keep retrying to get a working connection for some period of time

Reply:
Done